include/flatbuffers/flatbuffers.h

```diff
@@ -343,6 +343,11 @@ template<typename T> static inline size_t VectorLength(const Vector<T> *v) {
 struct String : public Vector<char> {
   const char *c_str() const { return reinterpret_cast<const char *>(Data()); }
   std::string str() const { return std::string(c_str(), Length()); }
+  #ifdef FLATBUFFERS_HAS_STRING_VIEW
```

Comment:
Do these need `clang-format`? I'm not sure how to find where it is needed or not.

Reply:
So then... it won't format the rest of the block? Is there some sort of "disable-next-line" flag, rather than disabling the whole block?

Reply:
I don't think so.

Reply:
OK, done in 57390e63